src/sqlancer/Main.java

```diff
@@ -265,6 +268,16 @@ public void logReduced(StateToReproduce state) {
             }
             try {
                 reduceFileWriter.write(sb.toString());
+
+                File symbolicLinkToReduceFile = new File(LOG_DIRECTORY, "current-reduce.log");
```

Comment:
it works on windows. But I haven't tested it on Linux.
Currently, The symbolic link file refer to the latest modified file, which means, if a.log is created before b.log, but is modified later, the symlink file will link to `a.log`. I'm not sure if it's appropriate.
Also, I'm not sure if the concurrent issues should be considered here.

Reply:
> Currently, The symbolic link file refer to the latest modified file, which means, if `a.log` is created before `b.log`, but is modified later, the symlink file will link to `a.log`. I'm not sure if it's appropriate.
> Also, I'm not sure if the concurrent issues should be considered here.

I'd prefer to create the link when file created, not modified, i.e. it will link to the recent created file.

Reply:
> I'd prefer to create the link when file created, not modified, i.e. it will link to the recent created file.

In this way the previous log file should be deleted first. I think `ensureExistsAndIsEmpty` function should be altered a bit so that the log file in the sub dictionary `reduce` could be removed.

Reply:
I see, you're right. I missed the other logs. So we might not need to create a separate link for the reduce log?